cef/help.md

```diff
@@ -1,25 +1,21 @@
 # Description
 
-Common Event Format (CEF) is an extensible, text-based format that defines a syntax for log records comprised of a
```

Comment:
There is some good information in the original that we should build off of. If anything keep that, and add your sentence after.

Reply:
Same for the others